content/en/blog/_posts/2025-01-11-Understanding-and-Visualizing-Kubernetes-Network-Policies:-A-Community-Perspective.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: blog
+title: "Visualizing Kubernetes Network Policies: Introducing knetvis"
```

Comment:
I don't know if the kubernetes blog should be used as a promoting of independent projects, we should consult what is the existing policies around this

Reply:
Genuine market surveys and comparisons: fine. Vendor pitches: not fine.

Reply:
Actually the article is not a vendor pitch @aojea - that title was a mistake, I'd recommit now. if you read through the article, it compares different k8s tools in network visualization